Sources/Actomaton/MainActomaton.swift

```diff
@@ -50,9 +50,7 @@ package final class MainActomaton2<Action, State>: MainActomaton
     where Action: Sendable, State: Sendable
 {
 #if os(Linux)
-    package var state: State {
-        actomaton.state
-    }
+    package private(set) var state: State
```

Comment:
@puliaiev
FYI this file change addresses https://github.com/Actomaton/Actomaton/pull/93 .

Reply:
wow, this is nice)